### ISSUE TITLE
Popup Modal is not accessible

### ISSUE THREAD

**Issue by SimonSiefke** [freeCodeCamp/freeCodeCamp#37949]
**Describe the bug**
The popup modal for becoming a sponsor cannot be closed by just using the keyboard.

**To Reproduce**
1. Do some challenges which triggers the modal

**Expected behavior**
- The modal closes when pressing the `Esc` key
- The modal closes when clicking somewhere outside of the modal
- I can navigate to the "Ask me later" button with the tab key to close the modal

**Screenshots**

![Screenshot from 2019-12-17 22-18-44](https://user-images.githubusercontent.com/23744935/71035470-d0d9f500-211b-11ea-99fc-159fc127e4c0.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: chrome and firefox
 - Version: Latest

**Smartphone (please complete the following information):**
Unknown

**Comment by raisedadead** [comment]
This is by design to the best of my knowledge. I would defer to Quincy for a confirmation.

**Comment by QuincyLarson** [comment]
@SimonSiefke Thanks for reporting that the "ask me later" link should be accessible by tab but you're correct - it isn't. We will fix this.

We decided not to have escape key / clicking outside of the modal to dismiss this modal (like we do with other modals) since the goal is to get people to actually slow down and consider donating to our nonprofit - not to just mash the escape key.
